app/code/Magento/Email/Model/Transport.php

```diff
@@ -112,4 +127,14 @@ public function getMessage()
     {
         return $this->message;
     }
-}
+
+    /**
+     * Retrieve store id
+     * @return int | null
+     */
+    public function getStoreId()
+    {
+        return $this->store ? $this->store->getId() : null;
```

Comment:
`$this->store` is populated in the constructor. Is there a reason to check for it additionally?